app/controllers/user_recipes_controller.rb

```diff
@@ -18,6 +18,12 @@ def index
           end
       end
 
+      def destroy
+        @user_recipe = UserRecipe.find(params[:id])
+        @user_recipe.destroy
+        render json: @user_recipe
+      end
+
```

Comment:
are there tests to cover this functionality? Also, what if the delete is unsuccessful? How will the user experience/be notified of that?